charts/atomic-app/values.yaml

```diff
@@ -13,6 +13,7 @@ nameOverride: ""
 fullnameOverride: ""
 
 configSecret: ""
+migrationConfigSecret: null
```

Comment:
I think a more common way to handle this would be to leave it commented out

Reply:
I commented it and added a comment